src/style.css

```diff
@@ -1,3 +1,79 @@
-body {
+:root {
+  --link-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+}
```

Comment:
The previous comment was only about `box-sizing: border-box;`
Don't apply anything except `box-sizing` to *. It's still inefficient for browser
to read your web document